config/reports/virusScanReport.js

```diff
@@ -0,0 +1,47 @@
+import { generateReportFromData } from '../helpers.js';
+import { querySudo as query } from '@lblod/mu-auth-sudo';
+
+export default {
+  cronPattern: '4 2 * * *',
+  name: 'virusScanReport',
+  execute: async () => {
+    const reportData = {
+      title: 'Virus Scan Report',
+      description:
+        'A list of files that are considered malicious, or where the status is unknown',
+      filePrefix: 'virus-scan-report',
+    };
+
+    const queryString = `
+PREFIX stix: <http://docs.oasis-open.org/cti/ns/stix#>
+PREFIX nfo: <http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#>
+PREFIX nie: <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#>
+
+SELECT DISTINCT ?file ?fileName ?scanResult ?scanDate WHERE {
+  GRAPH ?g {
+    ?analysis a stix:MalwareAnalysis ;
+      stix:analysis_started ?scanDate ;
+      stix:sample_ref ?file ;
+      stix:result ?scanResult .
+      FILTER (?scanResult IN ("malicious", "unknown"))
+
+      ?file nfo:fileName ?fileName .
+  }
+}
+ORDER BY DESC(?scanDate)`;
```

Comment:
Would it be more 'in your face' if you first orde by status (malicious on top) and then by date?

Reply:
So something like `ORDER BY ?scanResult DESC(?scanDate)`?

Reply:
yeah